library/Network/Wai/Cli.hs

```diff
@@ -88,13 +92,29 @@ instance Options WaiOptions where
 
 
 #ifdef WaiCliUnix
+
+fdStart :: CInt
```

Comment:
Please use UnicodeSyntax everywhere to be consistent with the rest of the file! `s/::/∷/`, `s/->/→/`, `s/<-/←/`